sale_timesheet_task_exclude/__init__.py

```diff
@@ -0,0 +1,4 @@
+# Copyright (C) 2023 Open Source Integrators
+# License AGPL-3.0 or later (http://www.gnu.org/licenses/agpl).
```

Comment:
Are changes of the Copyright necessary, even though you did not change any code?!?